money_bot/utils/db_utils.py

```diff
@@ -1,6 +1,22 @@
 from money_bot.utils import models
 
 
+async def is_user_in_db(user_id: int):
+    return bool(await get_user_by_id(user_id))
+
+
+async def set_referrer_id(user_id: int, referrer_id: int):
+    user = await get_user_by_id(user_id)
+    user.referrer_id = referrer_id
+    await user.commit()
+
+
+async def increase_money_amount(user_id: int, increase_value_amount: int):
```

Comment:
So if you added this function, you should use it everywhere, even in earn module